learntools/embeddings/nbs/2-exercises.ipynb

```diff
@@ -50,6 +50,10 @@
    "source": [
     "## Part 1: Generating Recommendations\n",
     "\n",
+    "\n",
+    "#### DB: I'm concerned that the user is learning more about how to use recommender systems in parts 1-2 below, rather than about factorization (which is nominally the topic of this lesson).\n",
```

Comment:
That's fair. My original train of thought for this exercise was that I would talk about why this is actually called "matrix factorization", and show how we can directly calculate the whole matrix of predicted user-item ratings (or a row thereof), by doing matrix multiplication of the learned embedding matrices. But at some point I decided that was too technical and not very useful.

I do think part 5 gives some interesting insight into the model.

I'm not sure what else we could do here. Again, there's the possibility of having them train a factorization model on another dataset (like goodreads book ratings).